src/plugins/kibana/public/settings/sections/indices/add_data_steps/paste_samples_step.js

```diff
@@ -1,12 +1,36 @@
-var modules = require('ui/modules');
-var template = require('plugins/kibana/settings/sections/indices/add_data_steps/paste_samples_step.html');
+const modules = require('ui/modules');
```

Comment:
Mind using `import` here? several prs are working on moving everything this way.